design/sketch.webidl

```diff
@@ -505,7 +504,9 @@ interface WebGPURenderPassEncoder : WebGPUProgrammablePassEncoder {
     void setScissorRect(u32 x, u32 y, u32 width, u32 height);
 
     void setIndexBuffer(WebGPUBuffer buffer, u32 offset);
-    void setVertexBuffers(u32 startSlot, sequence<WebGPUBuffer> buffers, sequence<u32> offsets);
+    // "startSlot" refers to an index into the "inputs" sequence in the currently bound pipeline's WebGPUInputStateDescriptor.
+    // "buffers" and "offsets" are parallel.
```

Comment:
We can enforce that they're parallel by making a new object which holds both items.

Reply:
If we did this, it would probably make sense to use it in setIndexBuffer, copyBufferToBuffer and possibly BufferCopyView and BufferBinding too.